lib/roku_builder/plugins/core.rb

```diff
@@ -59,6 +59,14 @@ def self.parse_options(parser:, options:)
       end
       parser.separator ""
       parser.separator "Other Options:"
+      parser.on("-L", "--console_log [FILE]", "Save monitor logs in file") do |l|
+        options[:logfile] = l || "config"
+        options[:logging] = true
+      end
+      parser.on("-C", "--clear_log [FILE]", "Clear saved monitor logs in file") do |l|
```

Comment:
can we make this param so that it doesn't take a filepath? A common usage for me would be that I need to try several times to get the specific log I am looking for. Using these as they are I would need to add the filepath twice. If the filepath on this option is removed then it would just be a modifier for the -L option. Is there a usecase where you would want to clear the log but not start logging again?